frontend/public/components/app.jsx

```diff
@@ -133,24 +134,26 @@ class App extends React.PureComponent {
       <>
         <Helmet titleTemplate={`%s · ${productName}`} defaultTitle={productName} />
         <ConsoleNotifier location="BannerTop" />
-        <Page
-          header={<Masthead onNavToggle={this._onNavToggle} />}
-          sidebar={
-            <Navigation
-              isNavOpen={isNavOpen}
-              onNavSelect={this._onNavSelect}
-              onPerspectiveSelected={this._onNavSelect}
-            />
-          }
-        >
-          <ConnectedNotificationDrawer
-            isDesktop={isDrawerInline}
-            onDrawerChange={this._onNotificationDrawerToggle}
+        <GuidedTourDrawer>
+          <Page
+            header={<Masthead onNavToggle={this._onNavToggle} />}
+            sidebar={
+              <Navigation
+                isNavOpen={isNavOpen}
+                onNavSelect={this._onNavSelect}
+                onPerspectiveSelected={this._onNavSelect}
+              />
+            }
           >
-            <AppContents />
-          </ConnectedNotificationDrawer>
-        </Page>
-        <CloudShell />
+            <ConnectedNotificationDrawer
+              isDesktop={isDrawerInline}
+              onDrawerChange={this._onNotificationDrawerToggle}
+            >
+              <AppContents />
+            </ConnectedNotificationDrawer>
+          </Page>
+          <CloudShell />
+        </GuidedTourDrawer>
         <ConsoleNotifier location="BannerBottom" />
```

Comment:
`ConsoleNotifier` should be a child of the `GuidedTourDrawer`

Reply:
done